rskj-core/src/main/java/co/rsk/config/RskSystemProperties.java

```diff
@@ -250,36 +252,62 @@ public List<ModuleDescription> getRpcModules() {
 
         List<ModuleDescription> modules = new ArrayList<>();
 
-        if (!configFromFiles.hasPath("rpc.modules")) {
+        if (!configFromFiles.hasPath(RPC_MODULES_PATH)) {
             return modules;
         }
 
-        List<? extends ConfigObject> list = configFromFiles.getObjectList("rpc.modules");
+        ConfigValue modulesConfig = configFromFiles.getValue(RPC_MODULES_PATH);
+        if (modulesConfig.valueType() == ConfigValueType.LIST) {
+            List<? extends ConfigObject> list = configFromFiles.getObjectList(RPC_MODULES_PATH);
+            modules = getModulesFromListFormat(list);
+        } else {
+            Map<String, Object> modulesMap = configFromFiles.getObject(RPC_MODULES_PATH).unwrapped();
```

Comment:
why do we need to "unwrap" an object here? It seems that `configFromFiles.getObject()` returns an instance of `ConfigObject`, which itself is `Map<String, ConfigValue>`, so that you can iterate over its keys

Reply:
I thought it was more readable like that, but what you are saying makes sense. I'll push the fix rn.

Reply:
`unwrapped()` returns `String-> Object` mapping, so in order to convert Object to Config you first "serialise" it to String and then parse it again to Config. I've just thought that this roundtrip could be omitted